.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```

Comment:
.

Reply:
I don't know how or why these random things happen

Reply:
I do - because you're not careful of what you commit (: 

If you just glance at the list of staged or edited files before you commit, this stuff should rarely happen.